File: Section Reports/Preview/CustomPreview/C#/CustomPreview/Reports/EmployeeSales.cs
using System;
namespace GrapeCity.ActiveReports.Samples.CustomPreview
{
	public partial class EmployeeSales : GrapeCity.ActiveReports.SectionReport
	{
		public EmployeeSales()
		{
			InitializeComponent();
			
			Document.Name = "Employee Sales Report";
			
			
		}
		private void EmployeeSales_ReportStart(object sender, EventArgs e)
		{
			
			chtSales.Titles[0].Text = "Sales by Employee for 1997";
			
			
		}
	}
}

File: Section Reports/Preview/CustomPreview/VB.NET/CustomPreview/Reports/EmployeeSales.vb
Public Class EmployeeSales
	Inherits GrapeCity.ActiveReports.SectionReport
	Public Sub New()
		MyBase.New()
		InitializeComponent()
		
		Document.Name = "Employee Sales Report"
		
		'
	End Sub
	Public ds As Data.OleDBDataSource
	Private Sub EmployeeSales_ReportStart(ByVal sender As System.Object, ByVal e As EventArgs) Handles MyBase.ReportStart
		
		chtSales.Titles(0).Text = "Sales by Employee for 1997"
		
		'
	End Sub
End Class
